KsApi/GraphMutation.swift

```diff
@@ -11,3 +11,9 @@ protocol GraphMutation: CustomStringConvertible {
 }
 
 public struct GraphMutationEmptyResponseEnvelope: Decodable {}
+
+extension GraphMutationInput where Self: Encodable {
+  public func toInputDictionary() -> [String: Any] {
+    return self.dictionaryRepresentation ?? [:]
+  }
+}
```

Comment:
Adds automatic conformance to `GraphMutationInput` when also confirming to `Encodable`, foresee any issues doing this?

Reply:
I think this is fine.

Reply:
<img src="https://media.giphy.com/media/z9AUvhAEiXOqA/giphy.gif"/>